src/main/resources/instructorEmailFragment-instructorCopyPreamble.html

```diff
@@ -1,5 +1,13 @@
 <p>
   The email below has been sent to students of course: [${courseId}] ${courseName}.
   <br><br>
+  Kindly note that this email simply serves as a preview of how the email will appear to the
+  students, and the link is not the actual link that the students will receive.
+  As such, please do not forward this email to students.
+  We recommend that you make the following announcement (edit content as you see fit) using an alternative means (e.g., your course announcements) to alert students:
+  <hr>
+  The TEAMMATES session ... in course ... is now open.
+  If you did not receive the unique access link via email, and you can't find it in your spam box either, go to ... (info about link recovery page).
```

Comment:
Lets provide them with the actual recovery page link as well
https://teammatesv4.appspot.com/web/front/help/session-links-recovery

Reply:
Added recovery page link.